app/internal/page/attorney/register.go

```diff
@@ -111,8 +111,10 @@ func Register(
 		YourAddress(logger, tmpls.Get("your_address.gohtml"), addressClient, lpaStore))
 	handleRoot(page.Paths.Attorney.ReadTheLpa, RequireLpa,
 		ReadTheLpa(tmpls.Get("attorney_read_the_lpa.gohtml"), lpaStore))
-	handleRoot(page.Paths.Attorney.RightsAndResponsibilities, RequireSession,
+	handleRoot(page.Paths.Attorney.RightsAndResponsibilities, None,
```

Comment:
I think this should `RequireSession`, or the continue button won't work. Also we already have a general rights and responsibilities page linked from the footer that sits outside the flows.